### DISCUSSION TITLE
i have an issue with the scaledobject it's in ready state but it's not scaling and

### DISCUSSION THREAD

**Discussion by Ranjith11903952** [kedacore/keda#4319] [Show and tell]
NAME     SCALETARGETKIND      SCALETARGETNAME   MIN   MAX   TRIGGERS   AUTHENTICATION   READY   ACTIVE   FALLBACK   AGE
engine                apps/v1.Deployment       engine                          1     50            redis      trigger                       True          False    False         37

READY is in True 
ACTIVE  is in Flase state is it normal or abnormol

there is celery in  my Djago application i have used scaledobjects but with redis it's not scaling could   you find me the solution

**Comment by tomkerkhove** [comment]
This is normaly - Basically it is saying that the SO is ready, but not active as trigger criteria has not been met.
